kubernetes-commands.el

```diff
@@ -145,6 +146,24 @@
 
 ;; Misc commands
 
+;;;###autoload
+(defun kubernetes-kill-buffers ()
+  "Kill all `kubernetes-mode' buffers."
+  (interactive)
+  (let* ((kubernetes-buffer-p
+          (lambda (buffer)
+            (let ((major-mode (buffer-local-value 'major-mode buffer)))
+              (or (eq major-mode 'kubernetes-mode)
+                  (eq (get-mode-local-parent major-mode) 'kubernetes-mode)))))
+         (buffers (-filter kubernetes-buffer-p (buffer-list)))
+         (num-buffers (length buffers)))
+    (if (not buffers)
+        (message "No Kubernetes buffers to kill.")
+      (when (y-or-n-p (format "Kill %s Kubernetes buffer(s)? " num-buffers))
```

Comment:
Is there a way to kill force kill buffers if one passes an argument. Say `C-u Q`

Reply:
Certainly.

Reply:
Done.